dist/tools/shellcheck/check.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2018 Freie Universität Berlin
+#                    Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+SHELLCHECK_CMD="$(which shellcheck)"
+export SHELLCHECK_OPTS="-e SC1090"
+
+if tput colors &> /dev/null && [ "$(tput colors)" -ge 8 ]; then
+    CERROR=$'\e[1;31m'
+    CRESET=$'\e[0m'
+else
+    CERROR=
+    CRESET=
+fi
+
+DIST_TOOLS=${RIOTBASE:-.}/dist/tools
+
+. "${DIST_TOOLS}/ci/changed_files.sh"
+
+FILES=$(FILEREGEX='(?=*.sh$)' changed_files)
+
+if [ -z "${FILES}" ]
+then
+    exit 0
+fi
+
+${SHELLCHECK_CMD} --version &> /dev/null || {
+    printf "%s%s: cannot execute \"%s\"!%s\n" "${CERROR}" "$0" "${SHELLCHECK_CMD}" "${CRESET}"
+    exit 1
+}
+
+ERRORS=$("${SHELLCHECK_CMD}" --format=gcc ${FILES})
```

Comment:
This line is reporting the following error for me: `dist/tools/shellcheck/check.sh:38:43: note: Double quote to prevent globbing and word splitting. [SC2086]`